include/zephyr/data/navigation.h

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2023 Trackunit Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DATA_NAVIGATION_H_
+#define ZEPHYR_INCLUDE_DATA_NAVIGATION_H_
+
+#include <zephyr/types.h>
+
+/**
+ * @brief Navigation utilities
+ * @defgroup navigation Navigation
+ * @ingroup utilities
+ * @{
+ */
+
+/**
+ * @brief Navigation data structure
+ *
+ * @details The structure describes the momentary navigation details of a
+ * point relative to a sphere (commonly Earth)
+ */
+struct navigation_data {
```

Comment:
There should probably be some DOP data in here, maybe VDOP HDOP and satellite count?

Reply:
HDOP is not relevant to the navigation utilities, it's related to the status of the GNSS. To get HDOP, use `gnss_get_info()`

Reply:
Got it, makes sense.